config/initializers/doorkeeper.rb

```diff
@@ -155,15 +155,15 @@
   # a registered application
   # NOTE: you must also run the rails g doorkeeper:application_owner generator
   # to provide the necessary support
-  #
-  # enable_application_owner confirmation: false
+
+  enable_application_owner confirmation: false
 
   # Define access token scopes for your provider
   # For more information go to
   # https://github.com/doorkeeper-gem/doorkeeper/wiki/Using-Scopes
-  #
+  
   # default_scopes  :public
-  # optional_scopes :write, :update
+  # optional_scopes :read, :write
```

Comment:
This line's been changed, but it's still a comment. Is that intentional?

Reply:
Yes, I've started the task using scopes and then moved to `cancancan`